cmd/csaf_checker/processor.go

```diff
@@ -445,12 +450,15 @@ func (p *processor) rolieFeedEntries(feed string) ([]csaf.AdvisoryFile, error) {
 		var rolieDoc any
 		err = json.NewDecoder(bytes.NewReader(all)).Decode(&rolieDoc)
 		return rfeed, rolieDoc, err
-
 	}()
 	if err != nil {
 		p.badProviderMetadata.error("Loading ROLIE feed failed: %v.", err)
 		return nil, errContinue
 	}
+
+	if rfeed.CountEntries() == 0 {
+		p.badProviderMetadata.warn("No entries in %s", feed)
```

Comment:
Doesn't it make more sense to report that under the ROLIE feeds? (Requirement 15)? Just a question - if you agree, please change it; if not please explain.

Reply:
I agree, adressed by https://github.com/csaf-poc/csaf_distribution/pull/357/commits/f7ed10af7747d843aba87fa84f0252c4a6d68971